attestation-service/src/lib.rs

```diff
@@ -183,11 +190,20 @@ impl AttestationService {
             None => InitDataHash::NotProvided,
         };
 
-        let claims_from_tee_evidence = verifier
-            .evaluate(&evidence, &report_data, &init_data_hash)
-            .await
-            .map_err(|e| anyhow!("Verifier evaluate failed: {e:?}"))?;
-        info!("{:?} Verifier/endorsement check passed.", tee);
+        let mut claims_from_tee_evidence: Vec<(Tee, String, TeeEvidenceParsedClaim)> = Vec::new();
+        for tee in &tees {
+            let ev = evidence
+                .get(tee)
+                .ok_or(anyhow!("No evidence found for {:?}", tee))?;
+            let verifier = verifier::to_verifier(tee)?;
+            let claims = verifier
```

Comment:
Is this usable at the moment? For instance, there are no GPU verifiers, correct?

Reply:
This PR introduces the sample_device verifier, but otherwise we don't have any additional functionality yet.